polygonwm.c

```diff
@@ -2215,6 +2322,58 @@ roundcorners(Client *c)
 	/*And ends here.*/
     XFreePixmap(dpy, mask);
     XFreeGC(dpy, shape_gc);
+	XSync(dpy, False);
+}
+
+void
+setmountain(Client *c, int ion)
+{
+    Window w = c->win;
+    XWindowAttributes wa;
+    XGetWindowAttributes(dpy, w, &wa);
+
+    if(!XGetWindowAttributes(dpy, w, &wa)) {
+		return;
+	}
+
+    int width = (borderpx * 2) + wa.width;
```

Comment:
Where did this `2` come from? Is the boundary configurable?

Reply:
this is not an issue. 2 is for two sides of the window. borderpx is the no of pixels that should be taken as border and that multiplied by 2 gives the total no of pixels that should be added to the width.

Reply:
Okay makes sense